.github/workflows/validatePR.yml

```diff
@@ -9,20 +9,19 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: dotnet test Src/CSharpier.Tests/CSharpier.Tests.csproj -c release
-  test_stdin:
-    runs-on: ubuntu-latest
```

Comment:
Are you sure this is worth burning through our free build minutes at double rate?
If you prefer powershell over bash, you can probably upgrade to [powershell core 6](https://aka.ms/pscore6) which runs in both Linux and Windows.

If you're just unfamiliar with bash/Linux, we can have a discord session together :fox_face:

Reply:
From what I saw, we get unlimited minutes because it's a public repo. But it was super easy to switch over to powershell core and there is no real need to run it on windows.

I should probably learn bash some day, but I feel like there is too much else to do!